.github/workflows/rock-release-oci-factory.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
         id: update-releases
         if: steps.changed-files.outputs.all_changed_and_modified_files != ''
         run: |
-          all_tags="$(jq -r 'to_entries[] | .key' $GITHUB_WORKSPACE/oci-factory/oci/${{ inputs.rock-name }}/_releases.json)"
+          all_tags="$(jq -r 'to_entries[] | .key' $GITHUB_WORKSPACE/oci-factory/oci/${{ inputs.rock-name }}/_releases.json | sed 's/-22.04//')"
```

Comment:
Should we document the rationale for the sed?

Reply:
I think it's pretty straightforward, as "-22.04" is appended to all the tags in OCI Factory, but we need to strip it to compare semantic versions (because it's not part of the semantic version)